tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -23,6 +23,9 @@
 import tensorflow as tf
 
 
+tf.contrib.rnn.Load()
```

Comment:
I am not exactly sure why this is needed, I followed the code from:
tensorflow/contrib/tensor_forest/python/ops/training_ops.py

w/o it, I had a lot of problems getting the .so file to load properly
during runtime.
On Fri, May 6, 2016 at 3:15 PM, ebrevdo <notifications@github.com> wrote:

> In tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py
> <https://github.com/tensorflow/tensorflow/pull/2002#discussion_r62398199>:
>
> > @@ -23,6 +23,9 @@
> >  import tensorflow as tf
> >
> >
> > +tf.contrib.rnn.Load()
>
> does this not happen by default upon rnn's *init* ??
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2002/files/727e0c76bed212fdf240a221e577e1b1a86848d5#r62398199>
>